auth_service/user_management/claims_repository/core/claims.py

```diff
@@ -0,0 +1,52 @@
+# Copyright 2021 - 2022 Universität Tübingen, DKFZ and EMBL
+# for the German Human Genome-Phenome Archive (GHGA)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+"""
+Definitions and helper functions for validating user claims.
+"""
+
+from datetime import datetime
+from typing import Callable
+
+from ....config import CONFIG
+from ..models.dto import AuthorityLevel, Claim, VisaType
+
+__all__ = ["is_valid_claim", "is_data_steward_claim"]
+
+
+INTERNAL_SOURCE = CONFIG.organization_url
+DATA_STEWARD_ROLE = "data_steward"
+
+
+def is_valid_claim(claim: Claim, now: Callable[[], datetime] = datetime.now) -> bool:
```

Comment:
Sollen wir uns bei den Zeiten auf UTC einigen?

```suggestion
def is_valid_claim(claim: Claim, now: Callable[[], datetime] = datetime.utcnow) -> bool:
```

Reply:
Es sollte eigentlich reichen, für den Service TZ=UTC zu setzen, oder?

Reply:
Anwering myself: Nicht ganz. Damit ist nicht sichergestellt, dass alle Zeiten eine TZ haben. Dafür habe ich jetzt einen eigenen Subtyp DateTimeUTC von datetime hinzugefügt.